streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/kstream/ProducedTest.scala

```diff
@@ -37,15 +37,15 @@ class ProducedTest extends FlatSpec with Matchers {
     internalProduced.valueSerde.getClass shouldBe Serdes.Long.getClass
   }
 
-  "Create a Produced with timestampExtractor and resetPolicy" should "create a Consumed with Serdes, timestampExtractor and resetPolicy" in {
+  "Create a Produced with streamPartitioner" should "create a Produced with Serdes and streamPartitioner" in {
```

Comment:
Side cleanup (was originally copied from `ConsumedTest` but not updated correctly)